### ISSUE TITLE
Error: Draggable not working

### ISSUE THREAD

**Issue by Snehal-Tembare** [ceryle/SegmentedButton#33]
I have 4 segmented buttons and I have set that group as draggable:true. In this case when I tap on 4th button its works but when I tap on 1st button it will show draggable at both places.
![issue](https://user-images.githubusercontent.com/19224538/36296597-03df38ce-1314-11e8-99c5-515532433fdc.JPEG)

**Comment by Snehal-Tembare** [comment]
@Swavis No, I didn't get a solution. For that purpose, I used another approach.

**Comment by Swavis** [comment]
@Snehal-Tembare if you have other solution please let me know...

**Comment by Snehal-Tembare** [comment]
@Swavis Which I used in that case also dragging is not working as its original draggable functionality. However, I used a simple linear layout with some drawable to give an effect like original UI. In that linear layout, I took a button or you may use textview also. Following is the UI which I made using linear layout.
![1548911919893](https://user-images.githubusercontent.com/19224538/52032194-039bee80-2546-11e9-8434-e2ed8cf0dd3b.JPEG)
